pkg/stringutils/normalize.go

```diff
@@ -0,0 +1,23 @@
+package stringutils
+
+import (
+	"strings"
+)
+
+// sanitizeString cleans up invalid characters
+func sanitizeString(s string) string {
```

Comment:
Should we make this exportable and update `pkg/process/normalize/normalize.go` to use it so we don't have `sanitizeString` in multiple places.

Reply:
There is a slight difference that stops me doing this. The invalid character in processes are either at the head or tail of the string but in annotations, they can be in the middle. And the processes are performance critical path.